Running tests on a test flinger device
Loading environment variables
Using environment file: tf_snapd_cm3_refresh
Deleting test flinger data
Creating job for snapd using a device
Running testflinger client
Submitting job to testflinger
JOB_ID: b245d874-f84d-4ec0-af8a-c01796a82f5b
Print job: /var/snap/testflinger-cli/current/job.yaml 
job_queue: dragonboard
global_timeout: 36000
provision_data:
    channel: stable
test_data:
    test_cmds: |
        #!/bin/bash
        sudo apt update || ps aux | grep apt
        sudo apt install -y git curl sshpass
        git clone https://github.com/sergiocazzolato/snappy-qa-jobs.git
        (cd snappy-qa-jobs && git checkout master)
        git clone https://git.launchpad.net/snapd-vendor snapd
        (cd snapd && git checkout master)
        . snapd/tests/lib/external/prepare-ssh.sh "{device_ip}" "22" "ubuntu"
        . snappy-qa-jobs/scripts/utils/register_device.sh "{device_ip}" "22" "test" "ubuntu" "sergiocazzolato@yahoo.com.ar"
        . snappy-qa-jobs/scripts/utils/refresh.sh "{device_ip}" "22" "test" "ubuntu" "stable" "edge"
        . snappy-qa-jobs/scripts/utils/run_setup.sh "{device_ip}" "22" "test" "ubuntu" ""
        . snappy-qa-jobs/scripts/utils/get_spread.sh
        . snappy-qa-jobs/scripts/utils/run_spread.sh "{device_ip}" "22" "snapd" "external:ubuntu-core-16-arm-64:tests/" "" "tests/main/auto-refresh,tests/main/interfaces-content" "-v"
Showing job data
This job is currently waiting on a node to become available. You will see no further output until the job starts running.
*******************************************

* Starting setup phase on dragonboard-002 *

*******************************************

Cleaning up container if it exists...
Error response from daemon: No such container: dragonboard002

***********************************************

* Starting provision phase on dragonboard-002 *

***********************************************
Install python-logstash if you want to use logstash logging
2018-08-19 04:40:54,536 dragonboard_002 INFO: BEGIN provision
2018-08-19 04:40:54,536 dragonboard_002 INFO: Booting Master Image
2018-08-19 04:40:54,536 dragonboard_002 INFO: Making sure the master image is booted
2018-08-19 04:40:54,536 dragonboard_002 INFO: Checking if test image booted.
2018-08-19 04:40:55,512 dragonboard_002 INFO: Running ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ubuntu@10.101.51.12 sudo sgdisk -o /dev/mmcblk1
Warning: Permanently added '10.101.51.12' (ECDSA) to the list of known hosts.
Warning: The kernel is still using the old partition table.
The new table will be used at the next reboot or after you
run partprobe(8) or kpartx(8)
The operation has completed successfully.
2018-08-19 04:40:57,994 dragonboard_002 INFO: Running ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ubuntu@10.101.51.12 sudo sync
Warning: Permanently added '10.101.51.12' (ECDSA) to the list of known hosts.
2018-08-19 04:40:59,671 dragonboard_002 INFO: Running snmpset -c private -v1 10.101.48.13 .1.3.6.1.4.1.318.1.1.12.3.3.1.1.4.2 i 2
iso.3.6.1.4.1.318.1.1.12.3.3.1.1.4.2 = INTEGER: 2
2018-08-19 04:40:59,789 dragonboard_002 INFO: Running sleep 10
2018-08-19 04:41:09,820 dragonboard_002 INFO: Running snmpset -c private -v1 10.101.48.13 .1.3.6.1.4.1.318.1.1.12.3.3.1.1.4.2 i 1
iso.3.6.1.4.1.318.1.1.12.3.3.1.1.4.2 = INTEGER: 1
2018-08-19 04:41:19,939 dragonboard_002 INFO: Checking if master image booted.
2018-08-19 04:41:37,983 dragonboard_002 INFO: Error checking device state. Forcing reboot...
2018-08-19 04:41:47,994 dragonboard_002 INFO: Checking if master image booted.
2018-08-19 04:43:36,729 dragonboard_002 ERROR: Bad data passed for provisioning
Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 356, in provision
    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['sudo', 'ubuntu-image', '-c', 'stable', '/home/ubuntu/plars/dragon.model', '-o', 'snappy.img']' returned non-zero exit status 1

2018-08-19 04:43:36,730 dragonboard_002 ERROR: exception
Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 356, in provision
    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['sudo', 'ubuntu-image', '-c', 'stable', '/home/ubuntu/plars/dragon.model', '-o', 'snappy.img']' returned non-zero exit status 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 248, in eat
    return self._dispatch()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 298, in _dispatch
    result = ingredient.dispatch(self.context)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 152, in dispatch
    return self._dispatch(context, 0)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
    return self._dispatch_None(context, level, retval, command)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
    return self._dispatch(context, level + 1)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
    return self._dispatch_None(context, level, retval, command)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
    return self._dispatch(context, level + 1)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 165, in _dispatch
    retval = command.invoked(context)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/__init__.py", line 37, in wrapper
    return func(*args, **kwargs)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/__init__.py", line 44, in invoked
    device.provision()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 359, in provision
    raise ProvisioningError("Error copying system-user assertion")
devices.ProvisioningError: Error copying system-user assertion
Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 356, in provision
    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['sudo', 'ubuntu-image', '-c', 'stable', '/home/ubuntu/plars/dragon.model', '-o', 'snappy.img']' returned non-zero exit status 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/srv/testflinger-agent/dragonboard002/env/bin/snappy-device-agent", line 4, in <module>
    __import__('pkg_resources').run_script('snappy-device-agents==0.0.1', 'snappy-device-agent')
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/pkg_resources/__init__.py", line 719, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1504, in run_script
    exec(code, namespace, namespace)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/EGG-INFO/scripts/snappy-device-agent", line 25, in <module>
    main()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/snappy_device_agents/cmd.py", line 75, in main
    Agent().main()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/snappy_device_agents/cmd.py", line 71, in main
    return AgentCommandRecipe(self).main(argv, exit)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/recipes/__init__.py", line 119, in main
    retval = bowl.eat(argv)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 254, in eat
    self._dispatch_failed()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 310, in _dispatch_failed
    ingredient.dispatch_failed(self.context)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 248, in eat
    return self._dispatch()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/core.py", line 298, in _dispatch
    result = ingredient.dispatch(self.context)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 152, in dispatch
    return self._dispatch(context, 0)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
    return self._dispatch_None(context, level, retval, command)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
    return self._dispatch(context, level + 1)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 179, in _dispatch
    return self._dispatch_None(context, level, retval, command)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 208, in _dispatch_None
    return self._dispatch(context, level + 1)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/guacamole-0.9.2-py3.5.egg/guacamole/ingredients/cmdtree.py", line 165, in _dispatch
    retval = command.invoked(context)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/__init__.py", line 37, in wrapper
    return func(*args, **kwargs)
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/__init__.py", line 44, in invoked
    device.provision()
  File "/srv/testflinger-agent/dragonboard002/env/lib/python3.5/site-packages/snappy_device_agents-0.0.1-py3.5.egg/devices/dragonboard/dragonboard.py", line 359, in provision
    raise ProvisioningError("Error copying system-user assertion")
devices.ProvisioningError: Error copying system-user assertion
complete
